runtime/Makefile

```diff
@@ -10,7 +11,7 @@ layers: export/console.zip export/php-72.zip export/php-73.zip export/php-72-fpm
 
 # The PHP runtimes
 export/php%.zip: build
-	PHP_VERSION=$$(echo $@| tail -c +8|head -c -5);\
+	PHP_VERSION=$$(echo $@ | cut -d'/' -f 2 | cut -d'.' -f 1);\
```

Comment:
`head -c -5` wasn't working on Mac, I tried using `cut` in the hopes that this is more portable.